install/overlays/vsphere/kustomization.yaml

```diff
@@ -36,8 +59,12 @@ secretGenerator:
 - name: peer-pods-secret
   namespace: confidential-containers-system
   literals:
-  - GOVC_USERNAME="" # set
-  - GOVC_PASSWORD="" # set
+  - GOVC_USERNAME=""   # set of the vCenter username is required.
```

Comment:
nit: `set of the vCenter` -> `set if the vCenter`

Reply:
Yeah same as below. There is no if. They must be set.